### ISSUE TITLE
Feature request: shortcuts for commands activate, instantiate, etc...

### ISSUE THREAD

**Issue by simonmandlik** [JuliaLang/Pkg.jl#3256]
We have `up` for `update`.

Would it be possible to add `act` for `activate`, `res` for `resolve` and possibly others?

When making use of the great Julia package manager to quickly switch between environments, I find that typing out these rather long commands slows me down, not to mention that I often make typos.

**Comment by fredrikekre** [comment]
`ac<TAB>`, `res<TAB>`.

What is your workflow if you use these commands so often that you feel they slow you down? I find that I very rarely use these when programming.

**Comment by simonmandlik** [comment]
Yeah, that would work too, thanks, though it's one more keystroke.

Most often I find myself typing out these commands when quickly testing or benchmarking same code in different versions of julia. I usually run `activate --temp` and then have to `resolve` and possibly `instantiate` (or even delete the `Manifest.toml` for earlier julia versions). I guess for proper extensive benchmarking it makes sense to setup these envs separately, but this became my go-to solution for quick testing like this over time.

**Comment by IanButterworth** [comment]
I don't think this is needed now we have tab completion hints. It's pretty self documenting.
